src/transformers/models/conditional_detr/configuration_conditional_detr.py

```diff
@@ -238,6 +238,19 @@ def num_attention_heads(self) -> int:
     def hidden_size(self) -> int:
         return self.d_model
 
+    def to_dict(self):
```

Comment:
This was missing here and in DeformableDetr to be able to serialize the config (or just print it) when the `bacjbone_config` is set.